company-profiles/carecru.md

```diff
@@ -0,0 +1,39 @@
+# CareCru Inc.
```

Comment:
The readme has `CareCru` but this titles the company with an Incorporated addition, which is correct?